rskj-core/src/main/java/co/rsk/net/sync/SyncConfiguration.java

```diff
@@ -134,4 +147,16 @@ public Duration getTimeoutWaitingSnapChunk() {
     public int getSnapshotSyncLimit() {
         return snapshotSyncLimit;
     }
+
+    public List<Node> getSnapshotTrustedPeers() {
+        if (snapshotTrustedPeers == null) {
```

Comment:
As mentioned above I'll do this check in the constructor, as it seems the snapshotTrustedPeers is not being updated anymore. So it is more efficient to do the check once instead of each time the method is checked.

Reply:
solved!